oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/IndexerSupport.java

```diff
@@ -76,12 +76,21 @@ public class IndexerSupport {
     private File indexDefinitions;
     private final String checkpoint;
     private File existingDataDumpDir;
+    private long minModified;
 
     public IndexerSupport(IndexHelper indexHelper, String checkpoint) {
         this.indexHelper = indexHelper;
         this.checkpoint = checkpoint;
     }
 
+    public long getMinModified() {
+        return minModified;
+    }
+
+    public void setMinModified(long minModified) {
+        this.minModified = minModified;
```

Comment:
oak document mentions:
`The _modified field contains an indexed low-resolution timestamp when the node was last modified. The time resolution is five seconds. This field is also updated when a branch commit modifies a node.
`
 If I am understanding it correctly anything updated till 4 sec will be set to 0 as per time resolution.
i.e  for a  14th second update  _modified property will be set to 10 sec.
For a  29th second update  _modified property will be set to 25 sec.

So to make sure that we don't miss any update 
I think minModified should be
`this.minModified = minModified - minModified/5`

@stefan-egli , @mreutegg am I correct?

Reply:
> anything updated till 4 sec will be set to 0

Anything updated on January 1st, 1970, will be set to 0.